main.go

```diff
@@ -284,6 +284,15 @@ func main() {
 		}
 	}
 
+	// TODO: feature gates?
```

Comment:
I don't think we need a new feature gate, we can have it under ROSA feature gate

Reply:
Yes. I did not mean a new feature gate here, just the existing rosa FG.